pkg/lib/files/files.go

```diff
@@ -31,6 +31,57 @@ import (
 	"github.com/cortexlabs/cortex/pkg/lib/errors"
 )
 
+func Open(path string) (*os.File, error) {
+	fileBytes, err := os.Open(path)
+	if err != nil {
+		return nil, errors.Wrap(err, ErrorReadFile(path).Error())
+	}
+
+	return fileBytes, nil
+}
+
+func OpenFile(name string, flag int, perm os.FileMode) (*os.File, error) {
+	file, err := os.OpenFile(name, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, perm)
+	if err != nil {
+		return nil, errors.Wrap(err, ErrorCreateFile(name).Error())
+	}
+
+	return file, err
+}
+func ReadFileBytes(path string) ([]byte, error) {
+	fileBytes, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, errors.Wrap(err, ErrorReadFile(path).Error())
+	}
+
+	return fileBytes, nil
+}
+
+func CreateFile(path string) (*os.File, error) {
+	file, err := os.Create(path)
+	if err != nil {
+		return nil, errors.Wrap(err, ErrorCreateFile(path).Error())
```

Comment:
Is `ErrorCreateFile` (and some of the other `files` errors) only used for wrapping, and if so should they be strings?